janusgraph-cql/pom.xml

```diff
@@ -33,6 +33,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>uk.org.lidalia</groupId>
+            <artifactId>slf4j-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Do you mean `version` tag or some other tag. If `version` than this dependency is added to the root `pom.xml`  with the version set. I.e.
```
<dependency>
    <groupId>uk.org.lidalia</groupId>
    <artifactId>slf4j-test</artifactId>
    <version>1.2.0</version>
    <scope>test</scope>
</dependency>
```

You can check that there are other loggers added below without version set as well (the version is set in the root `pom`):
```
<dependency>
    <groupId>uk.org.lidalia</groupId>
    <artifactId>slf4j-test</artifactId>
    <scope>test</scope>
</dependency>

<!-- Logging backends. -->
<!-- Enforce a classpath ordering constraint to ensure log4j is used for logging. -->
<dependency>
    <groupId>org.slf4j</groupId>
    <artifactId>slf4j-log4j12</artifactId>
    <optional>true</optional>
</dependency>
<dependency>
    <groupId>ch.qos.logback</groupId>
    <artifactId>logback-classic</artifactId>
</dependency>
<!-- End logging backends. -->
``` 

Overall we shouldn't enforce usage of any loggers when deploying to Sonatype but I guess it was added because we also provide janusgraph distributions. I didn't think about it yet but probably later we will need to think of excluding loggers implementations from deployed jars to maven and include logger dependencies when building janusgraph distributions. 
In this situation I just added a `test` logger dependency which won't be available in distributions (only for tests).

Reply:
Yes, I meant to say version tag. And you are right, I somehow missed the change in the root pom.xml (I didn't scroll down to the bottom).